src/clusterfuzz/_internal/cron/project_setup.py

```diff
@@ -808,11 +809,14 @@ def _sync_job(self, project, info, corpus_bucket_name, quarantine_bucket_name,
             project, info, template.engine, template.memory_tool,
             template.architecture)
       base_project_name = self._get_base_project_name(project)
+      oss_fuzz_project = ndb.Key(data_types.OssFuzzProject, project).get()
```

Comment:
do we need to add a check for OSS-Fuzz here, and only do this for OSS-Fuzz? otherwise we'll get None errors?

Reply:
i misread the code -- disregard.